x-pack/plugins/spaces/public/share_saved_objects_to_space/components/selectable_spaces_control.scss

```diff
@@ -1,3 +1,8 @@
 .spcShareToSpace__spacesList {
   margin-top: $euiSizeXS;
 }
+
+.spcShareToSpace__bottomLabel {
+  margin-top: 0 !important;
+  margin-bottom: 0 !important;
```

Comment:
I looked into this again to refresh my memory and figure out if/why I thought it was necessary at the time.

This is a result of us getting rid of `EuiFlyoutBody` so that we can stretch the SelectableSpacesControl to the bottom of the screen. However, we sometimes want to display additional text at the bottom of the flyout body, but _not_ in the footer. The `!important` is needed to override the top/bottom margin of the additional `EuiFlexGroup` that we are using to render the flyout 

Here's what it looks like currently, with `!important`:

![image](https://user-images.githubusercontent.com/5295965/122823937-f5cd9c00-d2ad-11eb-9c8d-329606630d8b.png)

Here's what it looks like if I remove `!important`:
![image](https://user-images.githubusercontent.com/5295965/122822928-ba7e9d80-d2ac-11eb-96fb-305bef2e9210.png)

If I change the closest `EuiFlexGroup` to use `gutterSize="none"`, it looks better:

![image](https://user-images.githubusercontent.com/5295965/122824142-3fb68200-d2ae-11eb-8dfa-e633c14443ea.png)

I think I'll use this instead so we can get rid of the `spcShareToSpace__bottomLabel` CSS class entirely. Thanks for bringing this up!

Reply:
Thanks for taking a second look! I like your approach of using gutterSize to control this